hit one timeout with `--repeat 3` going to try increasing the timeout and running it again
```
================================================================================
SESSION REPORT (ALL TESTS)
ducktape version: 0.7.6
session_id:       2020-02-28--011
run time:         41 minutes 34.257 seconds
tests run:        6
passed:           5
failed:           1
ignored:          0
================================================================================
test_id:    kafkatest.tests.core.throttling_test.ThrottlingTest.test_throttled_reassignment.bounce_brokers=False
status:     PASS
run time:   6 minutes 50.193 seconds
--------------------------------------------------------------------------------
test_id:    kafkatest.tests.core.throttling_test.ThrottlingTest.test_throttled_reassignment.bounce_brokers=True
status:     FAIL
run time:   4 minutes 54.465 seconds


    consumer was not assigned partitions within 60 seconds
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/ducktape/tests/runner_client.py", line 132, in run
    data = self.run_test()
  File "/usr/local/lib/python2.7/dist-packages/ducktape/tests/runner_client.py", line 189, in run_test
    return self.test_context.function(self.test)
  File "/usr/local/lib/python2.7/dist-packages/ducktape/mark/_mark.py", line 428, in wrapper
    return functools.partial(f, *args, **kwargs)(*w_args, **w_kwargs)
  File "/opt/kafka-dev/tests/kafkatest/tests/core/throttling_test.py", line 174, in test_throttled_reassignment
    lambda: self.reassign_partitions(bounce_brokers, self.throttle))
  File "/opt/kafka-dev/tests/kafkatest/tests/produce_consume_validate.py", line 97, in run_produce_consume_validate
    self.start_producer_and_consumer()
  File "/opt/kafka-dev/tests/kafkatest/tests/produce_consume_validate.py", line 53, in start_producer_and_consumer
    self.consumer.start()
  File "/usr/local/lib/python2.7/dist-packages/ducktape/services/service.py", line 234, in start
    self.start_node(node)
  File "/opt/kafka-dev/tests/kafkatest/services/console_consumer.py", line 297, in start_node
    self._wait_until_partitions_assigned(node)
  File "/opt/kafka-dev/tests/kafkatest/services/console_consumer.py", line 292, in _wait_until_partitions_assigned
    err_msg="consumer was not assigned partitions within %d seconds" % timeout_sec)
  File "/usr/local/lib/python2.7/dist-packages/ducktape/utils/util.py", line 41, in wait_until
    raise TimeoutError(err_msg() if callable(err_msg) else err_msg)
TimeoutError: consumer was not assigned partitions within 60 seconds

--------------------------------------------------------------------------------
test_id:    kafkatest.tests.core.throttling_test.ThrottlingTest.test_throttled_reassignment.bounce_brokers=False
status:     PASS
run time:   6 minutes 59.611 seconds
--------------------------------------------------------------------------------
test_id:    kafkatest.tests.core.throttling_test.ThrottlingTest.test_throttled_reassignment.bounce_brokers=True
status:     PASS
run time:   7 minutes 49.718 seconds
--------------------------------------------------------------------------------
test_id:    kafkatest.tests.core.throttling_test.ThrottlingTest.test_throttled_reassignment.bounce_brokers=False
status:     PASS
run time:   6 minutes 42.782 seconds
--------------------------------------------------------------------------------
test_id:    kafkatest.tests.core.throttling_test.ThrottlingTest.test_throttled_reassignment.bounce_brokers=True
status:     PASS
run time:   8 minutes 17.261 seconds
--------------------------------------------------------------------------------
```

--- a/tests/kafkatest/services/console_consumer.py
+++ b/tests/kafkatest/services/console_consumer.py
@@ -284,9 +284,13 @@ class ConsoleConsumer(KafkaPathResolverMixin, JmxMixin, BackgroundThreadService)
         jmx_tool.jmx_attributes = ["assigned-partitions"]
         jmx_tool.assigned_partitions_jmx_attr = "kafka.consumer:type=consumer-coordinator-metrics,client-id=%s:assigned-partitions" % self.client_id
         jmx_tool.start_jmx_tool(self.idx(node), node)
-        jmx_tool.read_jmx_output(self.idx(node), node)
         assigned_partitions_jmx_attr = "kafka.consumer:type=consumer-coordinator-metrics,client-id=%s:assigned-partitions" % self.client_id
-        wait_until(lambda: assigned_partitions_jmx_attr in jmx_tool.maximum_jmx_value,
+
+        def read_and_check():
+            jmx_tool.read_jmx_output(self.idx(node), node)
+            return assigned_partitions_jmx_attr in jmx_tool.maximum_jmx_value
+
+        wait_until(lambda: read_and_check(),
                 timeout_sec=timeout_sec,
                 backoff_sec=.5,
                 err_msg="consumer was not assigned partitions within %d seconds" % timeout_sec)

--- a/tests/kafkatest/tests/core/throttling_test.py
+++ b/tests/kafkatest/tests/core/throttling_test.py
@@ -53,7 +53,7 @@ class ThrottlingTest(ProduceConsumeValidateTest):
         # ensure that the consumer is fully started before the producer starts
         # so that we don't miss any messages. This timeout ensures the sufficient
         # condition.
-        self.consumer_init_timeout_sec =  20
+        self.consumer_init_timeout_sec =  60
         self.num_brokers = 6
         self.num_partitions = 3
         self.kafka = KafkaService(test_context,

